windows-apps-src/security/smart-cards.md

```diff
@@ -77,6 +77,9 @@ SmartCardProvisioning provisioning = await
 
 Once [**RequestVirtualSmartCardCreationAsync**](https://docs.microsoft.com/uwp/api/windows.devices.smartcards.smartcardprovisioning.requestvirtualsmartcardcreationasync) has returned the associated [**SmartCardProvisioning**](https://docs.microsoft.com/uwp/api/Windows.Devices.SmartCards.SmartCardProvisioning) object, the virtual smart card is provisioned and ready for use.
 
+>[!NOTE]
+>In order to create a VSC using a UWP, the user running the UWP must be a member of the administrators group. If the user is not a member of the administrators group, VSC creation will fail.

```

Comment:
This will require some wordsmithing. The documentation doesn't use the abbreviation "VSC". "UWP" is an adjective not a noun.

Reply:
I expanded out VSC in the change and added app following UWP. Do you have any concerns on the overall messaging of the note?

Reply:
You own the messaging since it's your team's feature. I'm just being an editor.

Reply:
@oldnewthing, thank you for noting those two issues. @mapalko, thanks for fixing them.